.env

```diff
@@ -1 +1,2 @@
-NOTION_TOKEN=secret_rrm731Cn5OVwp9iV89ck5fgZtMh94OfXRe7IBRI1m2h
\ No newline at end of file
+NOTION_TOKEN=secret_rrm731Cn5OVwp9iV89ck5fgZtMh94OfXRe7IBRI1m2h
```

Comment:
This file contains secrets and should not be under version control (guess we all overlooked this) 🙈

We should remove this file and follow [this guide](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) to remove secrets from the Git history.

Instead, setting these variables should be described in the documentation.

Reply:
I can take care of the production deployment and GH Actions (the Notion token is already set, the Measurement ID I will add tomorrow).

But for the Vercel deployment and on the dev deployment in Finland (which I'm not sure is ready), we will need @git-anurag-hub and @jemaltahir to do that.

Reply:
Hi @tanya17-05, I have added a `GA_MEASUREMENT_ID` secret for GitHub Actions. It's the `G-` version, not `GA_`. Let me know if you need something else.

As I said, Vercel might need this to be set up separately by @git-anurag-hub